discovery-infra/utils.py

```diff
@@ -8,8 +9,9 @@
 from retry import retry
 import consts
 from logger import log
+import libvirt
 
-VIRSH_LEASES_COMMAND = "virsh -q net-dhcp-leases"
+conn = libvirt.open('qemu:///system')
```

Comment:
Make `'qemu:///system'` a global variable and use it throughout your code

Reply:
conn is global.